staging/copy.sh

```diff
@@ -186,6 +187,7 @@ find "${CLIENT_REPO_TEMP}" -type f \( \
 
 echo "remove cyclical godep"
 rm -rf "${CLIENT_REPO_TEMP}/_vendor/k8s.io/client-go"
+rm -rf "${CLIENT_REPO_TEMP}/staging"
```

Comment:
Does "turned around" mean changing the order? Yeah, that's what I saw as well. I guess it's because godep doesn't know how to calculate the copy destination when GOPATH[0] is longer than GOPATH[1].

I'll add the comments.